sales.html

```diff
@@ -3,24 +3,11 @@
 <html>
```

Comment:
both of your html files have a space at the top before you declare the doc type, delete that, and make sure both your html files have proper indentation